ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/InsertsStreamHandler.java

```diff
@@ -157,9 +157,10 @@ private Void handleInsertSubscriberException(final Throwable t,
       }
       log.error("Failed to execute inserts", toLog);
       // We don't expose internal error message via public API
-      ServerUtils.handleError(routingContext.response(), 500, ErrorCodes.ERROR_CODE_INTERNAL_ERROR,
-          "The server encountered an internal error when processing inserts."
-              + " Please consult the server logs for more information.");
+      routingContext.fail(500,
```

Comment:
Can we use one of the enums for more descriptive status codes please, e.g. `HttpStatusCode.SERVER_ERROR` rather than `500`.

Reply:
Imho, I prefer explicit error codes rather than indirecting through _yet another 3rd party dependency_. Constants files are great for constants that might change in the future, but HTTP status codes are never going to change.
But.. in the interests of an easy life, I don't mind changing it.
We can't use the Jetty error code constants as Jetty is going away.
I guess we could org.apache.HttpStatus which is currently being dragged in anyway.